lib/paths-provider.js

```diff
@@ -191,12 +191,17 @@ export default class PathsProvider extends EventEmitter {
 
       // Calculate distance to file
       const distanceToFile = relativePath.split(path.sep).length
+      
+      const imgRegex = new RegExp(`.(jpg|jpeg|png)$`)
```

Comment:
```suggestion
      const imgRegex = new RegExp(`.(png|svg|jpg|jpeg|jfif|pjpeg|pjp|gif|apng|ico|cur)$`)
```

Based on https://www.w3schools.com/html/html_images.asp